cogs/yourlsClient/yourlsCmd.py

```diff
@@ -355,6 +366,36 @@ async def edit(self, ctx: Context, keyword: str, newLongUrl: str):
             )
             await ctx.send(f"Short URL {keyword} now points to {newLongUrl}")
 
+    @yourlsBase.command(name="search")
+    async def search(self, ctx: Context, searchTerm: str):
+        """Find a keyword.
```

Comment:
Would be nice to have a bit more clarity I believe.

```suggestion
        """Get a list of keywords that resemble `searchTerm`.
```

Reply:
Thanks, applied.